assets/prefabs/characters/testcharacter.prefab

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
IMO this has way more components than it needs to have. But I guess we can improve on that as a follow-up... I started looking into creating minimal test entities from scratch without prefabs.

Reply:
will mark this as resolved for now, will be done as a follow-up